{"uuid":"7dfa9b57-05a1-40f2-a518-aa7af884e711","historyId":"fd0b80860366c2b22a90215f8843609b","fullName":"tests.TestExcelUtility.dataDrivenLoginTest","labels":[{"name":"package","value":"tests.TestExcelUtility"},{"name":"testClass","value":"tests.TestExcelUtility"},{"name":"testMethod","value":"dataDrivenLoginTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"excel_automation"},{"name":"subSuite","value":"tests.TestExcelUtility"},{"name":"host","value":"INTELLIGENT"},{"name":"thread","value":"4016@INTELLIGENT.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testing login using data stored in Excel file","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.io.FileNotFoundException: D:\\SQA\\webdriver_java\\excel_automation\\.\\src\\test\\resources\\login_data.xlsx (The process cannot access the file because it is being used by another process)","trace":"java.lang.RuntimeException: java.io.FileNotFoundException: D:\\SQA\\webdriver_java\\excel_automation\\.\\src\\test\\resources\\login_data.xlsx (The process cannot access the file because it is being used by another process)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:57)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:803)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:728)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:911)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\r\n\tat org.testng.TestRunner.run(TestRunner.java:621)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.io.FileNotFoundException: D:\\SQA\\webdriver_java\\excel_automation\\.\\src\\test\\resources\\login_data.xlsx (The process cannot access the file because it is being used by another process)\r\n\tat java.base/java.io.FileOutputStream.open0(Native Method)\r\n\tat java.base/java.io.FileOutputStream.open(FileOutputStream.java:295)\r\n\tat java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:236)\r\n\tat java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:185)\r\n\tat org.apache.poi.openxml4j.opc.internal.FileHelper.copyFile(FileHelper.java:65)\r\n\tat org.apache.poi.openxml4j.opc.ZipPackage.closeImpl(ZipPackage.java:480)\r\n\tat org.apache.poi.openxml4j.opc.OPCPackage.close(OPCPackage.java:470)\r\n\tat org.apache.poi.ooxml.POIXMLDocument.close(POIXMLDocument.java:189)\r\n\tat org.apache.poi.xssf.usermodel.XSSFWorkbook.close(XSSFWorkbook.java:610)\r\n\tat utility.ExcelUtils.getRowCount(ExcelUtils.java:37)\r\n\tat tests.TestExcelUtility.getLoginData(TestExcelUtility.java:53)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:53)\r\n\t... 24 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1688374107291,"stop":1688374107299}